src/components/Icon.jsx

```diff
@@ -32,11 +32,24 @@ const SortIcon = () => {
     </div>
   );
 };
+const CloseIcon = () => {
+  return (
+    <button>

```

Comment:
You are only expected to return the svg, not the button. So that later you can do
```
<button onClick={clicked}>
    <CloseIcon />
</button>
```